.circleci/config.yml

```
@@ -30,3 +34,12 @@ workflows:
           matrix:
             parameters:
               tox_env: ["flake8", "mypy", "py37"]
+      - gcp-gcr/build-and-push-image:
+        context: data-eng-airflow-gcr
```

```suggestion
          context: data-eng-airflow-gcr
```

Looks like I mangled the whitespace here.